sdk-component-tests/webpack.config.js

```diff
@@ -0,0 +1,55 @@
+const path = require("path");
+const webpack = require("webpack");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+
+module.exports = {
+  mode: "development",
+  stats: "verbose", // This will make Webpack more verboseNODE
+  resolve: {
+    alias: {
+      "embedding-sdk": path.resolve(
+        __dirname,
+        "../resources/embedding-sdk/dist/main.bundle.js"
+      ),
+      // Make sure we use the same react version as the embedding-sdk
+      // without this, we would get a different version of react -> invalid hook calls
+      // or in some scenarios, even an error about "useMemo" not found on undefined
+      react: path.resolve(__dirname, "../node_modules/react"),
+      "react-dom": path.resolve(
+        __dirname,
+        "../node_modules/react-dom"
+      ),
+    },
```

Comment:
These lines are what made the stuff actually work, even when the tests were on a different folder (outside the core app), it was crashing with un-helpful errors.

Reply:
Oh that's smart, my initial POC was done on a folder outside of the metabase app so I had to do it with the resolve, but now that's inside it should work in that way too